app/controllers/admin/dashboard_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
     @statsdrafts = Article.drafts.where('created_at > ?', today).count
     @statspages = Page.where('published_at > ?', today).count
     @statuses = Note.where('published_at > ?', today).count
-    @statuserposts = Article.published.where('published_at > ?', today).count(conditions: { user_id: current_user.id })
+    @statuserposts = current_user.articles.published.where('published_at > ?', today).count
```

Comment:
this was to troubleshoot admin error - couldn't load admin page earlier.. should've done this on separate branch!